tests/foreman/ui/test_hostcollection.py

```diff
@@ -626,3 +627,24 @@ def test_positive_remove_package_group(self):
             for package in FAKE_0_CUSTOM_PACKAGE_GROUP:
                 self._validate_package_installed(
                     self.hosts, package, expected_installed=False)
+
+    @tier3
+    def test_positive_install_errata(self):
```

Comment:
>Is only one host present in hostcollection?

Take a look [here](https://github.com/SatelliteQE/robottelo/pull/3666/files/61d8e566c9d33bddc69153f1136b7f7688240d0a#diff-9907e07842190758b07fb14214cffc72L413) and at [setUp](https://github.com/SatelliteQE/robottelo/pull/3666/files/61d8e566c9d33bddc69153f1136b7f7688240d0a#diff-9907e07842190758b07fb14214cffc72R460). Currently we're creating 2 hosts, this number can be easily increased if needed.
> Anyway, do we have actual verification that errata was installed for all hosts in host collection?

We're verifying packages for each host in host collection. Take a look at [_validate_package_installed()](https://github.com/SatelliteQE/robottelo/pull/3666/files/61d8e566c9d33bddc69153f1136b7f7688240d0a#diff-9907e07842190758b07fb14214cffc72L485).